addons/website_payment/static/src/snippets/s_donation/000.scss

```diff
@@ -13,7 +13,7 @@
         outline: none;
         max-width: 145px;
         background-color: transparent;
-        font-size: inherit;
+        @include font-size(inherit);
```

Comment:
I guess the mixin handle `inherit`, but really it would be better not to use it IMO.